src/actions/nodePoolActions.js

```diff
@@ -142,13 +142,6 @@ export function nodePoolPatch(clusterId, nodePool, payload) {
           nodePool,
         });
 
-        new FlashMessage(
```

Comment:
Removed it from here because it was showing up when failing to scale a node pool. Node pool scaling has error displaying built-in, so we don't need the additional flash message